wallet/src/de/schildbach/wallet/service/CoinJoinService.kt

```diff
@@ -489,6 +509,8 @@ class CoinJoinMixingService @Inject constructor(
         // remove all listeners
         mixingCompleteListeners.forEach { coinJoinManager?.removeMixingCompleteListener(it) }
         sessionCompleteListeners.forEach { coinJoinManager?.removeSessionCompleteListener(it) }
+        clientManager.stopMixing()
+        coinJoinManager?.stop()
```

Comment:
This will shutdown mixing completely, and set the masternodeGroup to null.  The masternodeGroup does not want to be started, stopped and restarted, so we destroy it.